spec/controllers/api/drops_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Api::DropsController, type: :controller do
+
```

Comment:
This controller test is currently empty and therefore "adds no value" to the codebase. Since you have proper request specs, I tend to suggest dropping it entirely. But you could also consider filling it with examples, if you like.